### DISCUSSION TITLE
Is a new barrier version coming out

### DISCUSSION THREAD

**Discussion by wokawoka** [debauchee/barrier#1919] [General]
Since the last barrier version was released more than 1,5yrs ago, is a new version of barrier planned for the foreseeable future?
(I also see 29 pull requests at the moment)
Thank you!

**Comment by tonymynd** [comment]
Hi wokawoka, barrier has been a blessing for me, use with windows 10 and several Linux distros, including amazing Debian buster on a super old Acer1. How can I donate money to the project?

**Reply by shymega** [reply]
Barrier is no longer maintained. See 'Input Leap'. We haven't setup funding yet.

**Reply by NicolasGoeddel** [reply]
That's a bummer.
